services/QuillLMS/app/queries/quill_big_query/query.rb

```diff
@@ -12,8 +12,12 @@ def run
       raise NotImplementedError
     end
 
+    def post_query_transform(query_result)
+      query_result
+    end
+
     def run_query
-      runner.execute(query)
+      post_query_transform(runner.execute(query))
```

Comment:
I like standardizing the post_query_transform in one place.  For this reason, I think it would be a good idea to update Thomas has a post_process method [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/queries/admin_diagnostic_reports/diagnostic_aggregate_query.rb#L37), that way the approach has some momentum.